services/core-web/src/components/common/RenderTime.js

```diff
@@ -44,6 +44,7 @@ const RenderDate = (props) => (
       {...props.input}
       placeholder={props.placeholder}
       onChange={props.input.onChange}
+      onBlur={() => {}}
```

Comment:
If you'd rather have onBlur be a prop and the default value be `() => {}` and have `onBlur={props.onBlur}` let me know 🙂